apps/storybook/.storybook/main.js

```diff
@@ -1,37 +1,9 @@
-const path = require('path')
-
-const reactComponentRoot = path.resolve(__dirname, '../../../packages')
-
 module.exports = {
   addons: [
-    '@storybook/addon-a11y/',
-    '@storybook/addon-actions/',
-    '@storybook/addon-knobs/',
-    '@storybook/addon-viewport/',
+    '@storybook/addon-essentials',
+    '@storybook/addon-a11y',
+    'storybook-addon-performance/register',
+    'storybook-addon-jsx',
   ],
-  presets: [
-    {
-      name: '@storybook/addon-docs/preset',
-      options: {
-        configureJSX: true,
-      },
-    },
-  ],
-  stories: ['../../../packages/**/src/**/*.stories.js'],
-  webpackFinal: async (config, { configType }) => {
-    const rushCIPath = process.env.RUSH_TEMP_FOLDER
-
-    const mdxJsxRule = config.module.rules.find(
-      (rule) => rule && rule.include && rule.include[0]
-    )
-
-    mdxJsxRule && mdxJsxRule.include.push(reactComponentRoot)
-    mdxJsxRule && mdxJsxRule.exclude.push(/\/node_modules\//)
-
-    if (mdxJsxRule && rushCIPath) {
-      mdxJsxRule.include.push(path.resolve(rushCIPath))
-    }
-
-    return config
-  },
+  stories: ['../../../packages/*/src/**/*.stories.js'],
```

Comment:
Should this be `packages/**/src`? double star

Reply:
this was very deliberate, actually. each of our packages is exactly 2 folders deep, and this precludes it from finding stories in the node_modules of workspace projects (since local projects are linked)